crates/libs/core/src/client/mod.rs

```diff
@@ -109,6 +119,7 @@ impl FabricClientBuilder {
         Self {
             sn_handler: None,
             cc_handler: None,
+            conf_pkg_handler: None,
```

Comment:
Where is this passed to SF? Seems like it is dropped after builder is dropped.

Reply:
It wasn't. Added a wrapper to handle it.

Reply:
But said wrapper isn't complete or functional. Trying to wrap my head around the interaction of Rust lifetime rules and COM....